clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/LoadBalancerInstance.java

```diff
@@ -29,6 +30,21 @@
 @NoArgsConstructor
 public class LoadBalancerInstance {
   String id;
+  String uid;
   String zone;
   Map<String, Object> health;
+
+  public LoadBalancerInstance(String id, String zone, Map<String, Object> health) {
```

Comment:
This form of the constructor is used by `TitusTargetGroupServerGroupProvider`

Reply:
:+1: